UI/data/themes/Yami.qss

```diff
@@ -196,8 +196,7 @@ SourceTree::item:selected:hover {
     color: palette(text);
 }
 
-QListWidget::item:disabled,
-QListWidget::item:hover {
+QListWidget::item:disabled {
```

Comment:
I wonder if the original intent was something closer to 

```
QListWidget::item:disabled,
QListWidget::item:disabled:hover {
```

Reply:
I have a vague recollection that @Warchamp7 ran into some issues on Windows where QSS behaved strangely without certain combinations or selectors that otherwise did not make sense. As with any UI changes, some before/after images would be helpful to reviewing this.

Reply:
Updated to include all possible combinations of the :disabled and :hover

Reply:
Background should only be transparent for disabled items. I think this was originally a fix for the Add Existing source list, where the hover color was appearing even when the list was disabled.